src/store/middlewares/peers.js

```diff
@@ -16,20 +15,20 @@ const peersMiddleware = store => next => (action) => {
   // else default network
   if (loginNetwork) {
     loginNetwork = loginNetwork.slice(-1).shift();
-  } else if (!loginNetwork) {
-    loginNetwork = liskCoreUrl ? networks.customNode : networks.default;
+  } else if (!loginNetwork && liskCoreUrl) {
+    loginNetwork = { ...networks.customNode, address: liskCoreUrl };
+  } else if (!loginNetwork && !liskCoreUrl) {
+    loginNetwork = networks.default;
```

Comment:
Wouldn't be better to use a ternary here? Instead of having another `else if`. Like:
```
loginNetwork = liskCoreUrl ? { ...networks.customNode, address: liskCoreUrl } : networks.default
```